SIL.Windows.Forms/ImageToolbox/PalasoImage.cs

```diff
@@ -288,10 +288,53 @@ public static PalasoImage FromFile(string path)
 				Metadata = Metadata.FromFile(path),
 				_tempFilePath = tempPath
 			};
+			NormalizeImageOrientation(i.Image);
```

Comment:
I tried that. The resulting image looks right in Explorer. I didn't actually check whether it looks right in Bloom, because it did NOT look right in the image toolbox.
http://www.daveperrett.com/articles/2012/07/28/exif-orientation-handling-is-a-ghetto/ indicates that browser support for this flag is very patchy. So hard-rotating may also make our books more portable across browsers (and eventually e-book readers).